docs/developer/deployment.rst

```diff
@@ -50,3 +50,35 @@ Mantid Framework Deployment Procedure
   When running ``conda build`` locally, ensure that ``conda-build`` is up to date (``conda update conda-build``). This can be a source of difference between what is observed on the CI (install fresh into clean conda env) and a potentially stale local environment. You should also ensure that the channel order specified is the same as is applied in the CI for your ``conda build`` command. Refer to order applied in ``conda build`` step in pipeline yaml file. Priority decreases from left to right in your command line argument order. You should also ensure that your local `~/.condarc` file does not prioritise any unexpected/conflicting channels and that flag settings such as ``channel_priority: false`` are not utilised. Note that you can set ``--override-channels`` to your ``conda build`` command to prevent local `.condarc` files getting getting in the way.
 
 
+Documentation on Github Pages
+-----------------------------
+
+The procedure below describes the steps to follow in order to publish documentation pages from a new repository in the Scipp organisation to the scipp.github.io website.
+
+In this example, we assume that the new repository is named ``ess`` (the full path including organisation is hence ``scipp/ess``).
+
+#. Clone the ``ess`` repository: ``git clone git@github.com:scipp/ess.git``
+
+#. Create a `gh-pages` branch on the repository with ``git checkout -b gh-pages``
+
+#. On Github, go to the repository ``Settings > Pages`` (you will need admin rights to see the ``Settings`` tab). You should see that it is now saying "Your site is published at ``https://scipp.github.io/ess/``". It should also be saying it is using the ``gh-pages`` branch and the ``/root`` folder.
+
+#. On your local machine, generate a ssh key, without an email nor passphrase: ``ssh-keygen -t rsa -b 4096``. When prompted for a file name, use ``azure-ess-key`` (where ``ess`` is the name of the github repository).
+
+#. Go once again to the repository's ``Settings > Deploy keys``. Add a new key, and copy the contents of the ``azure-ess-key.pub`` file. Remember to also give the key ``write`` access to the repository.
```

Comment:
You previously only said anything about the Settings/Pages. Just strange as you are implying this is the not the first time you've been to the Deploy keys page.